### PR TITLE
implement old shutdown experience

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

File: examples/oldShutdownExperience.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+//  The new interface client shuts down too quickly... this script is for people who are nostalgic for the old experience.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+Script.scriptEnding.connect(function(){
+    while(true) {
+        // do nothing... but stop from shutting down.
```
Comment (huffman) [inline examples/oldShutdownExperience.js]: Did you want to add random memory allocation here?

Comment (ZappoMan) [inline examples/oldShutdownExperience.js]: good point...

Comment (huffman) [inline examples/oldShutdownExperience.js]: I think we could improve the Old Shutdown Experience ™ even more by randomly displaying alert windows here.

**Comment by problem** [comment]
ROFL.

**Comment by huffman** [comment]
spot on :+1:

**Comment by Atlante45** [comment]
I wish there was a like button for PRs

**Comment by ZappoMan** [comment]
Sadly, I found a bug in this version of the script... it turns out... it hangs on shutdown after a massive memory leak. Closing for now until we can fix it.

**Pull request closed without merging**